app/controllers/keywords_controller.rb

```diff
@@ -7,7 +7,7 @@ def index
     pagy, keywords_list = pagy(keywords)
 
     render locals: {
-      pagy: pagy, keywords: keywords_list
+      pagy: pagy, keywords: KeywordsDecorator.new(keywords_list)
```

Comment:
Based on our conventions, decorator pattern should be only used for the backend and presenter pattern should be used in view. So I think it will be presenter instead of decorator as we are using it in the view

```Ruby
pagy: pagy, keywords_collection_presenter: KeywordsCollection.new(keywords)
```

For reference, you can check this [controller](https://github.com/nimblehq/mia-web/blob/development/engines/mia_campaign/app/controllers/mia_campaign/campaign_influencers_controller.rb#L13) and [presenter](https://github.com/nimblehq/mia-web/blob/development/engines/mia_admin/app/presenters/mia_admin/withdrawals/campaign_influencer_collection_presenter.rb) in MIA project


![Screen Shot 2021-06-29 at 10 28 54 AM](https://user-images.githubusercontent.com/6076762/123732926-d98cb900-d8c4-11eb-89b7-5c8419d17212.png)

Reply:
Nice catch! I did not pay attention at the small warning in decorator as I thought `SimpleDelegator` was something totally separated from the Decorators part. 
Code updated & pushed @junan  :)